asim: skip more asim tests under CI

wenyihu6 opened the pull request:
Fixes: https://github.com/cockroachdb/cockroach/issues/153824
Epic: none 

---- 

**asim: skip more asim tests under CI**

This commit skips certain long-running asim tests in CI to reduce
kv/kvserver/asim/tests test time.

---
**asim: bump timeout to 15 min**

This commit bumps tests timeout to 15 min since tests under
asim/tests tend to take longer.

wenyihu6 committed:
asim: skip more asim tests under CI

This commit skips certain long-running asim tests in CI to reduce
kv/kvserver/asim/tests test time.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/153901)
<!-- Reviewable:end -->

wenyihu6 commented:
I'm not sure how comfortable we are with closing this failure with this PR. It is a little odd that it only started flaking recently - perhaps some of our recent asim changes slowed things down.

wenyihu6 committed:
asim: bump timeout to 15 min

This commit bumps tests timeout to 15 min since tests under
asim/tests tend to take longer.

tbg reviewed (approved):
LGTM. I'm comfortable blaming these test failures on s390x.
bors r+
